src/lib/containers/download_list_v2/DownloadListTable.tsx

```diff
@@ -68,19 +70,33 @@ export default function DownloadListTable(props: DownloadListTableProps) {
     }
   }, [isError, newError, handleError])
 
+  const allRows = data?.pages.flatMap(page => page.page) ?? []
+
   useEffect(() => {
+    const copyAllSynapseIDs = () => {
+      const synIDs = allRows.map((item: DownloadListItemResult)=>{
+        return item.fileEntityId
```

Comment:
I think it's crucial to also copy the version number (especially if the file isn't the latest version, but I think we should probably just always have it), or else that info will be lost when the user copies and pastes this list. The version number should likely also be shown in the table in the synId or its own column.